src/main/java/org/ecocean/MetricsBot.java

```diff
@@ -256,6 +272,31 @@ public static void refreshMetrics(String context) {
         else if(contributorsLabels.endsWith(",")) {contributorsLabels="\""+contributorsLabels.substring(0,(contributorsLabels.length()-1))+"\"";}
         csvLines.add(buildGauge("SELECT count(this) FROM org.ecocean.User WHERE username == null", "wildbook_datacontributors_total","Number of public data contributors",context,contributorsLabels));
         
+        //WBIA Metrics pull
+        String metricsURL = IA.getProperty(context, "IBEISIARestUrlAddImages");
+        if(metricsURL!=null) {
+          metricsURL=metricsURL.replaceAll("/api/image/json/","")+"/metrics";
```

Comment:
fyi, `IBEISIA.iaURL()` does this exact thing; just for future reference. no harm no foul.